roles/openshift_health_checker/openshift_checks/logging/fluentd_config.py

```diff
@@ -0,0 +1,144 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+from openshift_checks.ovs_version import OvsVersion
+
+
+class FluentdConfig(LoggingCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        logging_deployed = get_var(task_vars, "openshift_hosted_logging_deploy", default=False)
+
+        # determine OpenShift version
+        try:
+            ovs = OvsVersion(execute_module="")
+            version = float(ovs._get_openshift_version(task_vars))
+        except ValueError:
+            # if failed to parse OpenShift version, perform check anyway (if logging enabled)
+            return logging_deployed
+
+        return logging_deployed and version < 3.6
```

Comment:
Using floats to compare versions is buggy. Version-wise, 3.10 > 3.6, but numeric-wise 3.10 < 3.6. Suggestion: use a version tuple.